content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: documentation
+title: Upgrading Jenkins Java version from 8 to 11
+---
+
+In order to upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11, there are a few details and steps to take care.
+
+== Backup
+
+As for any upgrade, we recommend you to backup the `JENKINS_HOME`.
+It is also recommended to test the upgrade with the backup before performing it on the production instance.
+
+== Upgrading Plugins
```

Comment:
There is no section about updating Jenkins masters and Java themselves. E.g. do you recommend to update from 2.107.x/Java8 to 2.164/Java11 straight away? Or do you recommend to update Jenkins first or then Java? How to upgrade Java after all? How to do eerything in Docker and so on?

Reply:
That's actually an excellent point: I haven't thought of that. Will do.